website/docs/install/partials/_quickstart-run-execution-node.md

```diff
@@ -41,6 +42,15 @@ import TabItem from '@theme/TabItem';
                 <TabItem value="ipc"><pre><code>Nethermind.Runner --config sepolia --JsonRpc.Enabled true --HealthChecks.Enabled true --HealthChecks.UIEnabled true --JsonRpc.IpcUnixDomainSocketPath=/path/to/&lt;your.ipc&gt; --Merge.TerminalTotalDifficulty 17000000000000000</code></pre></TabItem>
             </Tabs>
       </TabItem>
+      <TabItem value="holesky">
+        <Tabs className='tabs-hidden-in-jwt-guide' groupId="protocol" defaultValue="jwt" values={[
+            {label: 'JWT', value: 'jwt'},
+            {label: 'IPC', value: 'ipc'}
+            ]}>
+                <TabItem value="jwt"><pre><code>Nethermind.Runner --config holesky --JsonRpc.Enabled true --HealthChecks.Enabled true --HealthChecks.UIEnabled true --JsonRpc.JwtSecretFile=/path/to/jwt.hex --Merge.TerminalTotalDifficulty 17000000000000000</code></pre></TabItem>
```

Comment:
hope this is the case for nethermind. probably fine to keep it this way not sure if any particular release should be mentioned.

Reply:
Oh.. it might not be. Let me check. I think TTD is not required

Reply:
Removed it for now. We might want to revisit these once https://github.com/NethermindEth/nethermind/issues/6033 is complete